_chapters/chapter_07.md

```diff
@@ -19,7 +19,7 @@ layout: rails_tutorial
 
 {% steps %}
 {% list %}
-  1.  Download the stylesheet file `bookstore.scss` [here](link-to-file.com).
+  1.  Download the stylesheet file `bookstore.scss` [here]({{site.baseurl}}/assets/downloads/bookstore.scss).
```

Comment:
While working on Chapter 13, I updated the stylesheet. It's a non-breaking change, but that's the kind of drift I'm worried about.